lib/widgets/lightbox.dart

```diff
@@ -208,11 +213,244 @@ class _LightboxPageState extends State<_LightboxPage> {
+                _controller!.value.position.formatHHMMSS(),
+                style: const TextStyle(color: Colors.white),
+              ),
+              Expanded(
+                child: Slider(
+                  value: _controller!.value.position.inSeconds.toDouble(),
+                  max: _controller!.value.duration.inSeconds.toDouble(),
+                  activeColor: Colors.white,
+                  onChanged: (value) {
+                    _controller!.seekTo(Duration(seconds: value.toInt()));
+                  },
+                ),
+              ),
+              Text(
+                 _controller!.value.duration.formatHHMMSS(),
+                style: const TextStyle(color: Colors.white),
+              ),
+            ],
+          ),
+          IconButton(
+            onPressed: () {
+              if (_controller!.value.isPlaying) {
+                _controller!.pause();
+              } else {
+                _controller!.play();
+              }
+            },
+            icon: Icon(
+              _controller!.value.isPlaying
+                ? Icons.pause_circle_rounded
+                : Icons.play_circle_rounded,
+              size: 50,
+            )),
+        ]));
+    }
+
+    return Theme(
+      data: themeData.copyWith(
+        iconTheme: themeData.iconTheme.copyWith(color: appBarForegroundColor)),
+      child: Scaffold(
+        backgroundColor: Colors.black,
+        extendBody: true, // For the BottomAppBar
+        extendBodyBehindAppBar: true, // For the AppBar
+        appBar: appBar,
+        bottomNavigationBar: bottomAppBar,
+        body: MediaQuery(
+          // Clobber the MediaQueryData prepared by Scaffold with one that's not
+          // affected by the app bars. On this screen, the app bars are
+          // translucent, dismissible overlays above the pan-zoom layer in the
+          // Z direction, so the pan-zoom layer doesn't need avoid them in the Y
+          // direction.
+          data: MediaQuery.of(context),
+
+          child: GestureDetector(
+            behavior: HitTestBehavior.translucent,
+            onTap: _handleTap,
+            child: SafeArea(
+              child: Center(
+                child: Stack(
+                  alignment: Alignment.center,
+                  children: [
+                    if (_controller != null && _controller!.value.isInitialized)
+                      AspectRatio(
+                        aspectRatio: _controller!.value.aspectRatio,
+                        child: VideoPlayer(_controller!)),
+                    if (_controller == null || !_controller!.value.isInitialized || _controller!.value.isBuffering)
+                      const SizedBox(
+                        width: 32,
+                        height: 32,
+                        child: CircularProgressIndicator(color: Colors.white))
+                  ]),
+              ))))));
+  }
+}
+
+extension DurationFormatting on Duration {
+  String formatHHMMSS() {
```

Comment:
This name is a bit misleading, because the method's behavior is that the presence of HH is conditional.

Rather than try to give this a name that accurately conveys what it does in a way that applies in a general `Duration` context, let's just embrace that it's a private helper for video playback, and make it a private static method on `_VideoLightboxPageState`. Then it can have a generic name like `_formatDuration`.